scaladoc/src/dotty/tools/scaladoc/site/StaticSiteContext.scala

```diff
@@ -71,6 +71,19 @@ class StaticSiteContext(
     orphanedFiles.flatMap(p => loadTemplate(p.toFile, isBlog = false))
   }
 
+  lazy val redirectTemplates: Seq[LoadedTemplate] = {
+    def doFlatten(t: LoadedTemplate): Seq[LoadedTemplate] =
+      t +: t.children.flatMap(doFlatten)
+    val mainFiles = templates.flatMap(doFlatten)
+    mainFiles.flatMap { loadedTemplate =>
+      loadedTemplate.templateFile.settings.getOrElse("page", Map.empty).asInstanceOf[Map[String, Object]].get("redirectFrom").map { case redirectFrom: String =>
+        val fakeFile = new File(docsPath.toFile, redirectFrom)
+        val redirectTo = fakeFile.toPath.getParent.relativize(loadedTemplate.file.toPath).toString.stripSuffix(".md") + ".html"
```

Comment:
Does that mean that we can’t redirect from a directory, like e.g. `/foo/bar/`?

Reply:
I assumed that we can redirect from `html` but cat change that, though I would need some test cases which paths should be valid

Reply:
Looking at the diff, I can’t tell whether this discussion is resolved or not. What’s its status? I recommend adding more tests.